From 83a52a082b4a85ded434184bbec89b4dae657953 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Mon, 30 Jul 2018 13:32:23 +0100
Subject: [PATCH] MANTA-3795 nginx build broken on recent platform images
 Reviewed by: Robert Mustacchi <rm@joyent.com> Approved by: Dan McDonald
 <danmcd@joyent.com>

---
 deps/nginx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/nginx b/deps/nginx
index de2e377..f054a7d 160000
--- a/deps/nginx
+++ b/deps/nginx
@@ -1 +1 @@
-Subproject commit de2e377384c2b4d61eed75bbd289d07558ab0bbc
+Subproject commit f054a7df29b99932cbe50c1216d545029b2e769c
-- 
2.21.0

